src/index.html

```diff
@@ -6,8 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <div class="product card__product">
+        <img
+          class="product__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27^ Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+        <div class="product__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product__code">
+          Product code: 195434
+        </div>
+
+        <div class="review product__review">
+          <div class="stars stars--4 review__stars">
```

Comment:
This classes is redundant
```suggestion
          <div class="stars stars--4">
```